src/main/treeDataProviders/treesManager.ts

```diff
@@ -35,6 +41,7 @@ export class TreesManager implements ExtensionComponent {
             vscode.window.registerTreeDataProvider('jfrog.xray.component', this._componentDetailsDataProvider),
             vscode.window.registerTreeDataProvider('jfrog.xray.issues', this._issuesDataProvider)
         );
+        this._storagePath = context.storagePath;
```

Comment:
I can't see a usage of storagePath. Are you sure you need it?

Reply:
Removed